test/integration/crud/remove.test.js

```diff
@@ -107,52 +107,6 @@ describe('Remove', function () {
     }
   });
 
-  it('should correctly remove only first document', {
```

Comment:
Similar to above, removing `single` means this test no longer passes.  To make this test pass, we would rework it to use `deleteOne`, but `deleteOne` is tested elsewhere.